sky/core.py

```diff
@@ -888,6 +891,80 @@ def spot_tail_logs(name: Optional[str], job_id: Optional[int],
     backend.tail_spot_logs(handle, job_id=job_id, job_name=name, follow=follow)
 
 
+@usage_lib.entrypoint
+def spot_cost_report(refresh: bool, verbose: bool) -> List[Dict[str, Any]]:
```

Comment:
Would be better to keep the argument align with the CLI. We can call it verbose for both CLI and here. Or call it `condensed` for both.